src/main/java/me/snowdrop/logging/TomcatLoggingApplication.java

```diff
@@ -48,6 +48,10 @@ public EmbeddedServletContainerFactory servletContainer() {
 
         // Default configuration file: logback-access.xml
         LogbackValve logbackValve = new LogbackValve();
+        
+        logbackValve.setFilename("logback-access.xml");
```

Comment:
This shouldn't be needed as logback-access.xml is already the default file name (see comment line 49)

Reply:
I tested without this line and it does not work.